src/seedu/addressbook/AddressBook.java

```diff
@@ -606,6 +615,26 @@ private static String executeListAllPersonsInAddressBook() {
     }
 
     /**
+     * Displays all persons in the address book to the user; in alphabetical order.
+     *
+     * @return feedback display message for the operation result
+     */
+    private static String executeListAllPersonsInAddressBookSorted() {
```

Comment:
also notice that your method name itself indicates you are doing multiple things at once...